config.py

```diff
@@ -21,6 +21,12 @@
 PASSWORD = ''  # App password
 EMAIL_SMTP = 'smtp.gmail.com:587'
 
+# sms authentication credentials
+URL = 'http://www.way2sms.com/api/v1/sendCampaign'
+APIKEY = '' # way2sms apiKey
```

Comment:
PEP8 naming conventions please :)
```suggestion
WAY2SMS_API_KEY = '' # way2sms apiKey
```